MarioAI-Base/src/main/java/ch/idsia/benchmark/mario/engine/sprites/Mario.java

```diff
@@ -58,7 +58,7 @@ public int getCode() {
 
 	}
 
-	private boolean isClone = false;
+	private static boolean isClone = false;
```

Comment:
This doesn't make sense. The point of isClone is to track wether the instance is a clone. If you run the simulation with an agent that calls clone even once, you will see that the entire simulation is broken. Enemies don't harm Mario and reaching the princess doesn't end the simulation :)